backend/src/govsg/models/govsg-template.ts

```diff
@@ -0,0 +1,47 @@
+import { Campaign } from '@core/models'
+import {
+  BelongsToMany,
+  Column,
+  DataType,
+  Model,
+  Table,
+} from 'sequelize-typescript'
+import { CampaignGovsgTemplate } from './campaign-govsg-template'
+
+@Table({ tableName: 'govsg_templates', underscored: true, timestamps: true })
+export class GovsgTemplate extends Model<GovsgTemplate> {
+  @Column({
+    type: DataType.BIGINT,
+    autoIncrement: true,
+    primaryKey: true,
+  })
+  id: number
+
+  @BelongsToMany(() => Campaign, {
+    through: { model: () => CampaignGovsgTemplate },
+  })
+  campaign!: Campaign
+
+  @Column({
+    type: DataType.TEXT,
+  })
+  body?: string
+
+  @Column({
+    type: DataType.ARRAY(DataType.STRING),
+    allowNull: true,
+  })
+  params?: Array<string>
+
+  @Column({
+    type: DataType.STRING,
+    unique: true,
+    allowNull: true,
+  })
+  whatsappTemplateLabel?: string | null
```

Comment:
same as previous

Reply:
think it might be better to call it `whatsappTemplateName`.

This is how the API call looks like:
<img width="337" alt="image" src="https://github.com/opengovsg/postmangovsg/assets/67887489/2048a22c-e27e-48ba-a8d3-af74a8e2a713">

Components is where we populate the parameters e.g. like this:
```
 "components": [{
   "type": "body",
   "parameters": [{
                "type": "text",
                "text": "name"
            },
            {
            "type": "text",
            "text": "Hi there"
            }]
      }] 
```

Think for consistency, we'll put these in the `params` column? Hmm

Reply:
maybe should [be unique](https://opengovproducts.slack.com/archives/C05AQ9K7R2T/p1687503185358169?thread_ts=1687485697.075039&cid=C05AQ9K7R2T)

Reply:
Hmm my first time seeing the namespace, but I think that one could be in the config as we will probably only use one namespace, but can change to name.

No it doesn't necessarily need to be unique so no not gonna make it unique.